.github/workflows/ci.yaml

```diff
@@ -25,6 +25,11 @@ jobs:
           ${{ runner.os }}-go-
     - name: Run unit tests
       run: make test-unit
+    - name: Remove generated code from report
+      run: |
+        grep -v .pb.go coverage.txt | grep -v zz_generated | grep -v service.connect.go > coverage.tmp
+        rm coverage.txt
```

Comment:
Pretty certain the `rm coverage.txt` line is unnecessary.

Reply:
You're right. Thanks!

Reply:
```suggestion
```